pkg/pool-weighted/contracts/lib/CircuitBreakerLib.sol

```diff
@@ -197,28 +197,29 @@ library CircuitBreakerLib {
             return 0;
         }
 
-        // Retrieve the weight complement passed in and bptPrice computed when the circuit breaker was set.
-        uint256 bptPrice = circuitBreakerState.decodeUint(_BPT_PRICE_OFFSET, _BPT_PRICE_WIDTH);
-        uint256 initialWeightComplement = circuitBreakerState.decodeUint(
-            _WEIGHT_COMPLEMENT_OFFSET,
-            _WEIGHT_COMPLEMENT_WIDTH
+        // Retrieve the weight passed in and bptPrice computed when the circuit breaker was set.
+        uint256 referenceBptPrice = circuitBreakerState.decodeUint(_BPT_PRICE_OFFSET, _BPT_PRICE_WIDTH);
```

Comment:
If we're calling the other 'references', I'd not call this one a ref. It's something else, not a precomputed value.

Reply:
This seems more like a true ref than the  others though.